doc/templates/command/template.cpp

```diff
@@ -0,0 +1,42 @@
+#include "template.hpp"
+
+#include <iostream>
+
+namespace dnf5 {
+
+using namespace libdnf::cli;
+
+void TemplateCommand::set_argument_parser() {
+    auto & ctx = get_context();
+    auto & parser = ctx.get_argument_parser();
+
+    auto & cmd = *get_argument_parser_command();
```

Comment:
Please add a comment to each block of the code - like // set short description for the command. it will be showed in help or something else.